manifests/a/AlignTechnology,Inc/InvisalignOutcomeSimulator/4.2.1.75/AlignTechnology,Inc.InvisalignOutcomeSimulator.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: AlignTechnology,Inc.InvisalignOutcomeSimulator

```

```suggestion
PackageIdentifier: AlignTechnology.InvisalignOutcomeSimulator
```
After changing this, the folder and files will need to be renamed